src/main/java/mate/academy/springboot/datajpa/service/ProductService.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.springboot.datajpa.service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.model.Product;
+
+public interface ProductService {
+    Product save(Product product);
+
+    Product getById(Long id);
+
+    void delete(Long id);
+
+    Product update(Product product);
+
```

Comment:
```suggestion
    Product save(Product product);

    Product getById(Long id);

    void delete(Long id);
```
use save() for both creating and updating()